README.md

```diff
@@ -26,6 +26,8 @@
 
 *New to Alda? You may be interested in reading [this blog post][alda-blog-post] as an introduction.*
 
+**Alda is a text-based programming language for music composition. It allows you to write and play back music using nothing but a text editor. The language's design equally favors aesthetics, flexibility and ease of use.**
```

Comment:
I appreciate the idea of adding this brief summary, it's just that I think the paragraph below is already a good (albeit slightly less brief) summary of what Alda is all about, and there is some duplication between the summaries.

Reply:
Thanks for pointing out my mistake and giving me a chance for hackoctoberfest